src/admin-views/category-colors/category-color-preview.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Template for displaying a category color preview in the admin table.
+ *
+ * @since TBD
+ *
+ * @var string $primary          The primary color hex value.
+ * @var string $secondary        The secondary color hex value.
+ * @var string $text             The text color hex value.
+ * @var string $priority         The priority of the category.
+ * @var string $hide_from_legend Whether the category is hidden.
+ * @var string $category_class   The category class identifier (e.g. tribe_events_cat-category-1).
+ */
+
+?>
+<span <?php tribe_classes( 'tec-events-taxonomy-table__category-color-preview', esc_attr( $category_class ) ); ?>
```

Comment:
You don't need `esc_attr()` when using `tribe_classes()`.

```suggestion
<span <?php tribe_classes( 'tec-events-taxonomy-table__category-color-preview', $category_class ); ?>
```